test_unstructured/documents/test_html.py

```diff
@@ -196,7 +336,6 @@ def test_parse_nothing():
 def test_read_with_existing_pages():
     page = Page(number=0)
     html_document = HTMLDocument.from_pages([page])
-    html_document._read()
```

Comment:
These calls are unnecessary and do not affect the outcome of the test. This method is called in due course simply by accessing `.pages` in the next line. Remove these as misleading the reader to think this call somehow changes state and is somehow required for test effectiveness.